packages/dapp/src/components/DualWidget/DualWidget.style.tsx

```diff
@@ -1,19 +1,162 @@
+  minHeight: showWelcome && '250px',
+  zIndex: showWelcome && 1500,
+
+  [`@media screen and (max-height: 490px)`]: {
+    display: showWelcome && 'none',
+
+    '& > .onramper-container, & .widget-wrapper': {
+      top: showWelcome && '50%',
+    },
+  },
+
+  '&:after': {
+    content: showWelcome && !showFadeOut && '" "',
+    background:
+      showWelcome && theme.palette.mode === 'dark'
+        ? 'linear-gradient(to top, #200a47 0%, transparent 100%)'
+        : showWelcome && theme.palette.mode === 'light'
+        ? 'linear-gradient(to top, #e8dafb 0%, transparent 100%)'
+        : 'transparent',
+    position: 'absolute',
+    height: '150px',
+    pointerEvents: 'none',
+    left: 0,
+    right: 0,
+    bottom: '-4px',
+    transitionProperty: showWelcome && 'height, bottom',
+    transitionDuration: showWelcome && '.3s',
+    transitionTimingFunction: showWelcome && 'ease-in-out',
+  },
+
+  '& .widget-wrapper, & > .onramper-container': {
+    height: showWelcome && !showFadeOut && '62%',
+    maxHeight: showWelcome && !showFadeOut && '350px',
+    position: showWelcome && !showFadeOut && 'absolute',
+    bottom: showWelcome && !showFadeOut && '0',
+    overflow: showWelcome && !showFadeOut && 'visible',
+  },
+
+  '& .widget-wrapper > div:before, & > .onramper-container:before': {
+    content: showWelcome && !showFadeOut && '" "',
+    position: 'absolute',
+    left: 0,
+    zIndex: 900,
+    cursor: 'pointer',
+    right: 0,
+    bottom: 0,
+    background:
+      showWelcome && !showFadeOut && theme.palette.mode === 'dark'
+        ? 'linear-gradient(180deg, rgba(243, 235, 255, 0) 0%, #000000 80%)'
+        : showWelcome && !showFadeOut && theme.palette.mode === 'dark'
+        ? 'linear-gradient(180deg, rgba(243, 235, 255, 0) 0%, #f3ebffd4 21.15%)'
+        : 'unset',
+    opacity: 0.5,
+    width: '392px',
+    margin: 'auto',
+    transitionProperty: showWelcome && 'opacity',
+    transitionDuration: showWelcome && '.3s',
+    transitionTimingFunction: showWelcome && 'ease-in-out',
+    borderTopRightRadius: '12px',
+    borderTopLeftRadius: '12px',
   },
-  [theme.breakpoints.up('md' as Breakpoint)]: {
-    paddingTop: theme.spacing(7),
+
+  '& > .onramper-container:before': {
+    height: 'calc( 100% - 24px )',
+    borderRadius: '12px',
+    opacity: 0.25,
+  },
+
+  '& > .onramper-container:hover:before': {
+    opacity: 0,
+    height: '100%',
+  },
+
+  '& > div > div:before': {
+    top: showWelcome && theme.spacing(6),
+    transitionProperty: showWelcome && 'top',
+    transitionDuration: showWelcome && '.3s',
+    transitionTimingFunction: showWelcome && 'ease-in-out',
+  },
+
+  '& > div > div:hover:before': {
+    opacity: 0,
+    top: showWelcome && 0,
+  },
+
+  '& > .widget-wrapper > div, & > .onramper-container': {
+    paddingTop: showWelcome && theme.spacing(6),
+  },
+
+  '& > div > div': {
+    top: showWelcome && theme.spacing(6),
+  },
+
+  '& > div > div, & > .onramper-container': {
```

Comment:
@chybisov As my CSS is very unspecific, I thought it´d be cool if we could pass a className to the widget´s container. Do you think this adds value to our users / integrators as well? I´d sure love to be more specific on my selectors as this looks odd and not clean